WireDomain/Sources/WireDomain/Repositories/TeamRepository.swift

```diff
@@ -72,6 +77,20 @@ final class TeamRepository: TeamRepositoryProtocol {
         await storeTeamLocally(team)
     }
 
+    func storeTeamMemberNeedsBackendUpdate(membershipID: UUID) async throws {
+        try await context.perform { [context] in
+
+            guard let member = Member.fetch(
+                with: membershipID,
+                in: context
+            ) else {
+                throw TeamRepositoryError.failedToFindTeamMember(membershipID)
+            }
+
+            member.needsToBeUpdatedFromBackend = true
```

Comment:
I think we should save here no?
In legacy code, it's not obvious when we save the context. But here I think the repository or its localstorage should have the responsability to save it

Reply:
fixed, thx